docs/quickstart.rst

```diff
@@ -237,6 +257,36 @@ credentials::
 
 Starting Everything Up
 **********************
+
+Match Shotgun users with JIRA users (for JIRA servers hosted by Atlassian only)
+===============================================================================
+
+.. code-block:: bash
+
+    $ python update_shotgun_users.py --settings <path to your settings.py> --project <id of your project>
+
+.. note::
+    For every user found in Shotgun, the script will search for a JIRA user with
+    the same email address. Note that if you have multiple users in Shotgun with
+    the same email address, only the first one, i.e. the one with the lowest id,
+    will be associated with a JIRA account.
+
+    If you wish to change the Shotgun user associated with a JIRA account, e.g. the
+    script associated the first Shotgun user with an account when you actually wanted
+    the second one, you can simply take the account id from the ``HumanUser.sg_jira_account_id``
+    field from one user and copy it to another user and then clear the original user's
+    account id.
+
+    If new users are added to JIRA and Shotgun, simply run this script again and the new user
```

Comment:
Suggestion: use of "simply" is unneeded. The sentence works fine without it.
It's also used in line 276 above as well.